examples/managed-identity/kubernetes-msi.json

```diff
@@ -42,8 +43,8 @@
       }
     },
     "servicePrincipalProfile": {
-      "servicePrincipalClientID": "",
-      "servicePrincipalClientSecret": ""
+      "servicePrincipalClientID": "msi",

```

Comment:
Is this an example or can you now actually specify msi as a value? If yes, seems confusing to me that id can be a real value or a string. If useManagedIdentity is true then this value does not need to be set.